server/chorus/routers/auth.py

```diff
@@ -107,15 +108,31 @@ async def register(user: User, db: Database):
 
 
 @router.post("/register/anonymous")
-async def register_anonymous(db: Database, settings: SettingsDep):
+async def register_anonymous(
+  db: Database, 
+  settings: SettingsDep, 
+  response: Response
+):
     anonymous_user = models.User(is_anonymous=True, session_id=uuid4())
     db.add(anonymous_user)
     db.commit()
 
     access_token = encode_access_token(
         {"sub": str(anonymous_user.session_id), "type": "anonymous"}, settings
     )
-    return {"access_token": access_token, "token_type": "bearer"}
+
+    response.set_cookie(
+        key="access_token",
+        value=access_token,
+        httponly=True,
+        secure=settings.cookie_secure,
+        samesite="none",
+    )
+    
+    return {
+      "message": "Anonymous user created",
+      "username": anonymous_user.session_id
```

Comment:
Do you think we should call this `session_id` transparently? Might be easier for the frontend I guess? Nitpick though!